action.yml

```diff
@@ -3,19 +3,31 @@ name: 'osv-scanner'
 description: 'Scans your directory against the OSV database (Experimental)'
 inputs:
   to-scan:
-    description: 'Directory to scan'
-    required: true
-    default: '/github/workspace'
-  version:
-    description: 'osv-scanner version to use'
+    description: 'Directories to scan'
+    required: false
+    default: "./"
+  results-file:
+    description: 'OUTPUT: Path to file to store results'
     required: true
-  arch:
-    description: 'osv-scanner architecture'
+  results-format:
+    description: 'Output result format'
+    required: false
+    default: 'sarif'
+  recursive-search:
+    description: 'Recursively scan though subdirectories'
+    required: false
```

Comment:
is `required: false` required if there is a `default` ?

Reply:
I don't think it's required, but I think it's clearer to explicitly state it for all for all inputs.

Reply:
Random note because frankly its weird imo: [`required` isn't actually enforced "yet"](https://github.com/actions/runner/issues/1070